tests/dags/common/test_slack.py

```diff
@@ -268,8 +270,8 @@ def test_send_with_context(http_hook_mock):
 def test_send_fails(http_hook_mock):
     s = SlackMessage()
     # Cause an exception within the raise_for_status call
-    http_hook_mock.return_value.run.return_value.raise_for_status.side_effect = (
-        Exception("HTTP Error 666")
+    http_hook_mock.run.return_value.raise_for_status.side_effect = Exception(
+        "HTTP Error 666"
```

Comment:
Very, very trival but I wonder if this should be clearer that it's a fake value for testing. It's probably fine 😅

Reply:
Ha, I can reword this message :slightly_smiling_face: